testutil/keeper/tokenomics.go

```diff
@@ -2,10 +2,11 @@ package keeper
 
 import (
 	"context"
+	"math"
 	"testing"
 
 	"cosmossdk.io/log"
-	"cosmossdk.io/math"
+	sdkmath "cosmossdk.io/math"
```

Comment:
nit: Any objection to `cosmosmath` to disambiguate between our sdk and cosmos-sdk?